tests/foreign_object/models/customers.py

```diff
@@ -1,56 +1,39 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-
-from __future__ import unicode_literals, print_function, absolute_import
-
-from django.db.models.deletion import CASCADE
-from django.db.models.fields.related import ForeignObject
 from django.db import models
+from django.db.models.fields.related import ForeignObject
+
 
 class Address(models.Model):
     company = models.CharField(max_length=1)
-    tiers_id = models.IntegerField()
-    city = models.CharField(max_length=255)
-    postcode = models.CharField(max_length=32)
+    customer_id = models.IntegerField()
 
     class Meta(object):
         unique_together = [
-            ("company", "tiers_id"),
+            ("company", "customer_id"),
         ]
 
 
 class Customer(models.Model):
     company = models.CharField(max_length=1)
     customer_id = models.IntegerField()
-    name = models.CharField(max_length=255)
     address = ForeignObject(
-        Address, on_delete=CASCADE, null=True,
```

Comment:
reordered the to_fields and from fields here instead of in Contact to properly the PhoneNumber model